lib/components/narrative/narrative-itineraries-header.tsx

```diff
@@ -175,13 +165,6 @@ export default function NarrativeItinerariesHeader({
               >
                 {pending ? searching : itinerariesFound}
               </h1>
-              {errors.length > 0 && (
-                <IssueButton onClick={onToggleShowErrors}>
```

Comment:
We don't need `onToggleShowErrors` anymore

Reply:
There should be an entire error view that can go too

Reply:
Remove `onToggleShowErrors` from the props.